mungegithub/mungers/release-note-label.go

```diff
@@ -92,14 +89,14 @@ func (r *ReleaseNoteLabel) EachLoop() error { return nil }
 // RegisterOptions registers options for this munger; returns any that require a restart when changed.
 func (r *ReleaseNoteLabel) RegisterOptions(opts *options.Options) sets.String { return nil }
 
-func (r *ReleaseNoteLabel) prMustFollowRelNoteProcess(obj *github.MungeObject) bool {
+func (r *ReleaseNoteLabel) prMustFollowRelNoteProcess(obj *github.MungeObject, comment bool) bool {
 	boolean, ok := obj.IsForBranch("master")
 
 	if !ok || boolean {
 		return true
 	}
 
-	parents := getCherrypickParentPRs(obj, r.config)
+	parents := getCherrypickParentPRs(obj, obj.Config())
```

Comment:
its not safe to use the config stored in `r` because `r` may not have been initialized if this func is called from the comment-deleter munger when release-note-label is not enabled.